### DISCUSSION TITLE
Terrible in small window

### DISCUSSION THREAD

**Discussion by guoyunhe** [community/community#55174] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![图片](https://github.com/community/community/assets/5836790/35e5a227-9344-40d2-8030-7aa60d052855)

**Comment by ooliver1** [comment]
I'll add to this, this happens on my desktop when using 20px font size. "Following" renders just fine but "For You" only recently has started overlapping the updates feed.

**Comment by shuaibird** [comment]
It seems the issue caused by `color-modes.scss`.
More specifically, `d-flex` or `width-full` leads to the layout breakage.

**Comment by thoughtlessmind** [comment]
It happens to me as well. 
CSS is really painful : )
<img width="566" alt="image" src="https://github.com/community/community/assets/51883613/f100203d-3ca7-410c-8581-2167229757cd">

**Comment by Shariar-Hasan** [comment]
i am also having the same problem :(
![image](https://github.com/community/community/assets/75330767/7d100887-929f-4a1e-9953-1113388fa2eb)
